### PR TITLE
[Doc enhancement][Correctcode and Addnote]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Corrected json section and added a note. Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/013e3f90e5a0f25c422fe2808c2f84c75eb2ad88/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2567062-9d51-4270-9e2f-2af72f23da49) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3a9783af7adb6afaa8998b541c2b4d1ca5e7444a/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/955ceaa7-acfe-4099-aad0-2f8a00407232) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c9a14824017c1251e1e8a4748a3c0035e77bacd/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/975add15-e5ac-43fd-a466-820361e15457) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ceb34aa51b440bc17be0b07f474b31399b5709ae/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84971c15-a96a-49f2-874e-ed69b4f0f10c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by leonyang12 — COMMENTED** [review]
We are in discussion with bot framework team on what should be in the "id" property of MeetingStartEndEventvalue object. You might want to update the doc based on the result.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa7b145859d67afd93ef2cb678eec51374d88923/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 90 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d86b841d-3089-4bd7-8de7-5a6b1ea540bd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -396,7 +396,7 @@ The following code provides an example of meeting start event payload:
     "value": { 
         "MeetingType": "Scheduled", 
         "Title": "Meeting Start/End Event", 
-        "Id":"meeting id", 
+        "Id": "conversation id",
```
[the remark this answers is not in the document]
Comment (Rajeshwari-v) [reply msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md]: @leonyang12,changed the "id" as "meeting id" as suggested by you. Please let me know for further updates in the doc.

**Review by leonyang12 — APPROVED** [review]
Looks good

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**